music21/pitch.py

```diff
@@ -474,73 +476,23 @@ def _greedyEnharmonicsSearch(oldPitches, scoreFunc=_dissonanceScore):
         newPitches.append(newPitch)
     return newPitches
 
-def simplifyMultipleEnharmonics(pitches, criterion=_dissonanceScore, keyContext=None):
-    r'''Tries to simplify the enharmonic spelling of a list of pitches, pitch-
-    or pitch-class numbers according to a given criterion.
-
-    A function can be passed as an argument to `criterion`, that is tried to be
-    minimized in a greedy left-to-right fashion.
-
-    >>> pitch.simplifyMultipleEnharmonics([11, 3, 6])
-    [<music21.pitch.Pitch B>, <music21.pitch.Pitch D#>, <music21.pitch.Pitch F#>]
-
-    >>> pitch.simplifyMultipleEnharmonics([3, 8, 0])
-    [<music21.pitch.Pitch E->, <music21.pitch.Pitch A->, <music21.pitch.Pitch C>]
-
-    >>> pitch.simplifyMultipleEnharmonics([pitch.Pitch('G3'),
-    ...                                    pitch.Pitch('C-4'),
-    ...                                    pitch.Pitch('D4')])
-    [<music21.pitch.Pitch G3>, <music21.pitch.Pitch B3>, <music21.pitch.Pitch D4>]
-
-    >>> pitch.simplifyMultipleEnharmonics([pitch.Pitch('A3'),
-    ...                                    pitch.Pitch('B#3'),
-    ...                                    pitch.Pitch('E4')])
-    [<music21.pitch.Pitch A3>, <music21.pitch.Pitch C4>, <music21.pitch.Pitch E4>]
-
-    The attribute `keyContext` is for supplying a KeySignature or a Key
-    which is used in the simplification:
-
-    >>> pitch.simplifyMultipleEnharmonics([6, 10, 1], keyContext=key.Key('B'))
-    [<music21.pitch.Pitch F#>, <music21.pitch.Pitch A#>, <music21.pitch.Pitch C#>]
-
-    >>> pitch.simplifyMultipleEnharmonics([6, 10, 1], keyContext=key.Key('C-'))
-    [<music21.pitch.Pitch G->, <music21.pitch.Pitch B->, <music21.pitch.Pitch D->]
-    
-    
-    Note that if there's no key context, then we won't simplify everything (at least
-    for now; this behavior may change, ).
-    
-    >>> pitch.simplifyMultipleEnharmonics([pitch.Pitch('D--3'),
-    ...                                    pitch.Pitch('F-3'),
-    ...                                    pitch.Pitch('A--3')])
-    [<music21.pitch.Pitch D--3>, <music21.pitch.Pitch F-3>, <music21.pitch.Pitch A--3>]
-
-
-
-    >>> pitch.simplifyMultipleEnharmonics([pitch.Pitch('D--3'),
-    ...                                    pitch.Pitch('F-3'),
-    ...                                    pitch.Pitch('A--3')],
-    ...                                    keyContext=key.Key('C'))
-    [<music21.pitch.Pitch C3>, <music21.pitch.Pitch E3>, <music21.pitch.Pitch G3>]
+def simplifyMultipleEnharmonics(pitches):
     '''
+    Replaces previous function with the new 'EnharmonicSimplifier' in analysis.
```

Comment:
No need for this line so long as it does what happened before.  But losing key-context is an issue -- the best enharmonics for B, E-, G- are likely to be different in C major vs. in Gb major.